ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -28,7 +28,7 @@ periodics:
         # prow doesn't allow init containers or a second container
         export PATH=$PATH:/tmp/bin
         mkdir /tmp/bin
-        curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.3/linux/oc.tar.gz | tar xvzf - -C /tmp/bin/ oc
+        curl https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | tar xvzf - -C /tmp/bin/ oc
```

Comment:
Shouldn't we use oc-4.1 to test OCP 4.1? CLI is a part of the product.

Reply:
This `oc` is not used in the tests, it's a hack to move the ISTags around and do the mirror. I'd prefer to use a new one so that fixes land in there.

Reply:
Yeah, I'm all for testing the product whenever possible, but here using an old `oc` does not have value. It's also one less place where we need to bump versions in handcrafted jobs on cuts.